sql/share/errmsg-utf8.txt

```diff
@@ -7962,3 +7962,5 @@ ER_NOT_ALLOWED_IN_THIS_CONTEXT
         eng "'%-.128s' is not allowed in this context"
 ER_DATA_WAS_COMMITED_UNDER_ROLLBACK
         eng "Engine %s does not support rollback. Changes were committed during rollback call"
+ER_PERIOD_PAIR_OF_SAME_COLUMNS
```

Comment:
We can't add new error codes in 10.4 and earlier. Better use some existing error message, like `ER_FIELD_SPECIFIED_TWICE`

Reply:
@FooBarrior Thank you for your comment. I've fixed along with your advice.